src/reference/asciidoc/logging-adapter.adoc

```diff
@@ -72,8 +67,8 @@ public class LoggingJavaApplication {
         return adapter;
     }
 
-    @MessagingGateway(defaultRequestChannel = "logChannel")
-    public interface MyGateway {
+   @MessagingGateway(defaultRequestChannel = "logChannel") // logChannel is created above via @ServiceActivator
```

Comment:
We don't need this comment here.
It's not responsibility of this chapter to describe that behavior.